src/sass/base/_hamburger.scss

```
@@ -0,0 +1,24 @@
+.header__hamburger {
+    margin-left: auto;
+}
+
+.hamburger__line {
+    display: block;
+    background-color: $color-dark;
+    height: 2px;
+    width: 40px;
+    margin: 6px auto;
+    transition: 0.2s;
+}
+
+.hamburger.active .hamburger__line:nth-child(2) {
+    opacity: 0;
+}
+
+.hamburger.active .hamburger__line:nth-child(1) {
+    transform: translateY(12px) rotate(45deg);
+}
+
+.hamburger.active .hamburger__line:nth-child(3) {
+    transform: translateY(-12px) rotate(-45deg);
+}
```

you can still utilize SCSS nesting for this
```suggestion
.hamburger.active {
  .hamburger__line {
    &:nth-child(1){
      transform: translateY(12px) rotate(45deg);
    }
    
    &:nth-child(2){
      opacity: 0;
    }
    
    &:nth-child(3){
      transform: translateY(-12px) rotate(-45deg);
    }
  }
}
```

👍🏻 for use of `transform`